man/systemd-analyze.xml

```diff
@@ -118,6 +118,13 @@
       <arg choice="plain">cat-config</arg>
       <arg choice="plain" rep="repeat"><replaceable>NAME</replaceable>|<replaceable>PATH</replaceable></arg>
     </cmdsynopsis>
+    <cmdsynopsis>
+      <command>systemd-analyze</command>
+      <arg choice="opt" rep="repeat">OPTIONS</arg>
+      <arg choice="plain">vercmp</arg>
```

Comment:
Nit: other verbs use the long format for the name rather than abbreviating, so what about `version-compare` for internal consistency?

Reply:
So… I made it `vercmp` to match `rpmdev-vercmp`, but I'm fine with the longer version too if people prefer that.

Reply:
If we want this to be part of our public api, and part of the image story, then to me it seems better to go for internal consistency - but I do not have a strong opinion, just an observation